packages/gestalt/src/Sheet.js

```diff
@@ -64,53 +113,60 @@ const DismissButton = ({
   />
 );
 
+/*
+
+<Sheet> component: the one with the actual component logic
+<SheetWithForwardRef> component: wrapper for forwarding ref to <Sheet>
+
+ */
 const SheetWithForwardRef: React$AbstractComponent<
-  Props,
+  SheetProps,
   HTMLDivElement
-> = forwardRef<Props, HTMLDivElement>(function Sheet(props, sheetRef): Node {
+> = forwardRef<SheetProps, HTMLDivElement>(function Sheet(
+  props,
+  sheetRef
+): Node {
   const {
     accessibilityDismissButtonLabel,
     accessibilitySheetLabel,
     children,
     closeOnOutsideClick = true,
-    onDismiss,
     footer,
-    heading,
+    heading = undefined,
```

Comment:
Default values are only triggered when the field is undefined, so these default values aren't doing anything.

Reply:
Yes, great catch! I only added them to keep flow happy, from the way I was using Union Types with Flow - but it seems it's no longer necessary, will remove it :)